easybuild/easyconfigs/m/Mesa/Mesa-10.4.5-intel-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '10.4.5'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True}
+
+sources = ['%sLib-%s.tar.gz' % (name, version)]
+source_urls = [
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+
+patches = ['Mesa-10.4.5_sse4_1.patch']
+
+pythonver = '2.7.9'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % ('Python', pythonver)
+
+dependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.2'),
+    ('makedepend', '1.0.5'),
+    ('Python', pythonver),
+    ('libxml2', '2.9.2', versionsuffix),
+    ('glproto', '1.4.17'),
+    ('libdrm', '2.4.59'),
+    ('Automake', '1.15'),
+    ('pkg-config', '0.28'),
+    ('libtool', '2.4.5'),
+    ('M4', '1.4.17')
+]
+
+osdependencies = [
+    'libX11-devel',  # Xlibs.h
+    'xorg-x11-proto-devel',  # X.h, glproto, xproto
+    'libXdamage-devel',
+    'libXext-devel',
+    'libXfixes-devel',
+]
```

Comment:
@hajgato: we have a Mesa easyconfig that doesn't have these OS deps, it would be nice to switch to that instead?